verl/single_controller/base/decorator.py

```diff
@@ -72,20 +72,48 @@ def init_predefined_execute_mode():
 init_predefined_execute_mode()
 
 
+def _chunk_tensordict(td: TensorDict, chunks: int) -> list[TensorDict]:
```

Comment:
Please move this function to verl/utils/tensordict_utils and add a test. Thanks!